plugin/collectd-setup

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# collectd-setup Startup script to setup collectd-mlab switch configuration.
+# chkconfig: - 85 16
+# description: collectd-setup performs auto-discovery of the local switch
+#   configuration to complete collectd setup. collectd-setup should run before
+#   collectd.
+
+
+# NOTE: Perform SNMP auto-discovery for switch configuration.
+# If we cannot update the snmp community string, then the script should halt.
+COLLECTD_SNMP="/etc/collectd-snmp.conf"
+COMMUNITY="/home/mlab_utility/config/snmp.community"
+SWITCH="s1.${HOSTNAME#*.}"
+
+function start() {
+
+    # If the community string file is missing or zero size.
+    if ! test -s "${COMMUNITY}" ; then
```

Comment:
To be sure, at this stage, disco_config.py will already have been run once to have generated $COMMUNITY? On the surface, it looks as if for some reason $COMMUNITY doesn't exist or is empty, that it will always stay that way without manual intervention. Most likely I'm just not understanding how the RPM install process works fully, or misreading this script.